.github/workflows/build.yml

```diff
@@ -270,3 +272,43 @@ jobs:
           repository: netgroup-polito/CrownOps
           event-type: release-event
           client-payload: '{"version": "${{ needs.configure.outputs.version }}"}'
+
+      # automatic helm repo management
+      # (from https://github.com/liqotech/liqo/blob/master/.github/workflows/integration.yml)
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.8.1
+
+      - name: Download chart releaser
+        run: |
+          curl -sSL "https://github.com/helm/chart-releaser/releases/download/v1.4.0/chart-releaser_1.4.0_linux_amd64.tar.gz" | tar -xz
+
+      - name: Package helm chart
+        run: |
+          # the output should be in the .cr-release-packages since cr index expects to find it there to create the helm index
+          helm package deploy/crownlabs --version "${{ steps.get_version.outputs.VERSION }}" --app-version "${{ steps.get_version.outputs.VERSION }}" --destination .cr-release-packages
+
+      - uses: ncipollo/release-action@v1
+        with:
+          artifacts: ".cr-release-packages/*"
+          generateReleaseNotes: true
+          token: ${{ secrets.CI_TOKEN }}
+          allowUpdates: true
+          tag: ${{ steps.get_version.outputs.VERSION }}
+          name: ${{ steps.get_version.outputs.VERSION }}
+          prerelease: ${{ steps.semver_parser.outputs.prerelease != '' }}
```

Comment:
Should be merged with the other step that creates the release. In liqo, there are two steps due to the fact that there are additional artifacts, which is not managed correctly by cr index.